easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,51 @@
+# by Ward Poelmans <wpoely86@gmail.com>
+
+easyblock = 'ConfigureMake'
+
+name = 'PLUMED'
+version = '2.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.plumed-code.org'
+description = """PLUMED is an open source library for free energy calculations in molecular systems which
+ works together with some of the most popular molecular dynamics engines. Free energy calculations can be
+ performed as a function of many order parameters with a particular  focus on biological problems, using
+ state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+ The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'usempi': 'True'}
+
+source_urls = ['https://github.com/plumed/plumed2/releases/download/v%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['f782d1eafffb919f27fcafc1b317c08b498ff51727ae20d15491f26ed332bbff']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('GSL', '2.5'),
+    ('libmatheval', '1.1.11'),
+    ('Python', '2.7.15'),
+]
+
+preconfigopts = 'env FC=$MPIF90 LIBS="$LIBLAPACK $LIBS" '
+configopts = ' --exec-prefix=%(installdir)s --enable-gsl --enable-modules=all --enable-python'
+prebuildopts = 'source sourceme.sh && '
+# install path for PLUMED libraries must be included in $LD_LIBRARY_PATH when Python bindings get built/installed
+preinstallopts = 'LD_LIBRARY_PATH="%(installdir)s/lib:$LD_LIBRARY_PATH" '
+
+sanity_check_paths = {
+    'files': ['bin/plumed', 'lib/libplumedKernel.%s' % SHLIB_EXT, 'lib/libplumed.%s' % SHLIB_EXT,
+              'lib/plumed/python/plumed.%s' % SHLIB_EXT],
+    'dirs': ['lib/plumed'],
```

Comment:
Redundant due to the checks in "files", drop it.

Reply:
same in the other plumed .eb

Reply:
Åke, not fully sure what you are after here.   You want me to drop the 

```
'dirs': ['lib/plumed'],
```

line?  Please confirm or explain better.

Ta
   Joachim

Reply:
Change to
'dirs': []